tests/model_fields/models.py

```diff
@@ -69,11 +69,18 @@ class WhizIterEmpty(models.Model):
 
 
 class Choiceful(models.Model):
+    class Suit(models.IntegerChoices):
+        DIAMOND = 1, _("Diamond")
+        SPADE = 2, _("Spade")
+        HEART = 3, _("Heart")
+        CLUB = 4, _("Club")
```

Comment:
I don't think translations are relevant here, or perhaps I'm missing something.
```suggestion
        DIAMOND = 1, "Diamond"
        SPADE = 2, "Spade"
        HEART = 3, "Heart"
        CLUB = 4, "Club"
```

Reply:
sure thing. It was just the canonical example from the other test.